commands.json

```
@@ -1503,6 +1503,208 @@
     "since": "3.2.0",
     "group": "geo"
   },
+  "GEOSEARCH": {
+    "summary": "Query a sorted set representing a geospatial index to fetch members matching a box or circular.",
+    "complexity": "O(N+log(M)), where N is the number of elements in the bounding box of the circular area or box area, the circular area is defined by the center and radius, the box area is defined by the center, height and width, and M is the number of items inside the index.",
+    "arguments": [
+      {
+        "name": "key",
+        "type": "key"
+      },
+      {
+        "command": "FROMMEMBER",
+        "name": "member",
+        "type": "string",
+        "optional": true
+      },
+      {
+        "command": "FROMLOC",
+        "name": [
+          "long",
+          "lat"
+        ],
+        "type": [
+          "double",
+          "double"
+        ],
+        "optional": true
+      },
+      {
+        "command": "BYRADIUS",
+        "name": [
+          "radius",
+          "unit"
+        ],
+        "type": [
+          "double",
+          "string"
+        ],
+        "optional": true
+      },
+      {
+        "command": "BYBOX",
+        "name": [
+          "width",
+          "height",
+          "unit"
+        ],
+        "type": [
+          "double",
+          "double",
+          "string"
+        ],
+        "optional": true
+      },
```

@itamarhaber do these work? (providing an array for `name` and `type`)
if not, i guess we should use something like this:
```suggestion
      {
        "type": "block",
        "name": "circle",
        "block": [
          {
            "command": "BYRADIUS",
            "name": "radius"
            "type": "double"
          },
          {
            "name": "unit",
            "type": "enum",
            "enum": [
              "m",
              "km",
              "ft",
              "mi"
            ]
          },
        "optional": true
      },
      {
        "type": "block",
        "name": "box",
        "block": [
          {
            "command": "BYBOX",
            "name": "width",
            "type": "double",
          },
          {
            "name": "height",
            "type": "double",
          },
          {
            "name": "unit",
            "type": "enum",
            "enum": [
              "m",
              "km",
              "ft",
              "mi"
            ]
          },
        "optional": true
      }
```

---

@itamarhaber if that's right, we need to apply that change to GEOSEARCHSTORE too